src/json_parser.cc

```diff
@@ -101,4 +102,50 @@ std::optional<bool> JSONParser::GetTopLevelBoolField(std::string_view field) {
   return value->BooleanValue(isolate);
 }
 
+std::optional<std::map<std::string, std::string>>
+JSONParser::GetTopLevelDictOfStrings(std::string_view field) {
```

Comment:
Is this using the recently added simdjson, or should it?

Reply:
No, this thign predates simdjson. We could do a separate PR to migrate the entire JSONParser to simdjson, but probably out of scope of this.